src/checkbox/src/Checkbox.tsx

```diff
@@ -197,6 +199,7 @@ export default defineComponent({
       mergedDisabled: mergedDisabledRef,
       renderedChecked: renderedCheckedRef,
       mergedTheme: themeRef,
+      labelId: `n-checbox-label-${createId(4)}`,
```

Comment:
The prefix may not be needed?

Reply:
Oh and I made a typo. I don't know we don't need it but might be more user friendly if we want to search for it in the devtool console